src/styles/clock.scss

```diff
@@ -0,0 +1,126 @@
+.main {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  width: 100%;
+  background-color: #123141;
+}
+
+.clock {
+  width: $clock-size;
+  height: $clock-size;
+  margin: 65px auto;
+  position: relative;
+  display: flex;
+  border: 30px solid $clock-color;
+  border-radius: 50%;
+  box-shadow: $clock-shadow;
+
+  &:before {
+    content: "";
+    width: $clock-point-size;
+    height: $clock-point-size;
+    position: relative;
+    margin: auto;
+    background: $clock-color;
+    border-radius: 50%;
+    z-index: 11;
+  }
+
+  &__hours {
+    position: absolute;
+    top: $clock-hand-position;
+    left: $clock-hand-position;
+    width: $clock-hand-size;
+    height: $clock-hand-size;
+    display: flex;
+    justify-content: center;
+    border-radius: 50%;
+    transform: rotate(180deg);
+    animation: rotateHours 43200s linear infinite;
+
+    &::before {
+      content: "";
+      width: 8px;
+      height: 120px;
+      background-color: #ff105e;
+      z-index: 10;
+      border-radius: $clock-hand-border;
+      box-shadow: $clock-hand-hour-shadow;
+    }
+  }
+
+  &__minutes {
+    position: absolute;
+    top: $clock-hand-position;
+    left: $clock-hand-position;
+    width: $clock-hand-size;
+    height: $clock-hand-size;
+    display: flex;
+    justify-content: center;
+    border-radius: 50%;
+    transform: rotate(180deg);
+    animation: rotateSeconds 3600s linear infinite;
```

Comment:
The animation for the minute hand is incorrect. Change 'rotateSeconds' to 'rotateMinutes' to use the correct animation.